kong/plugins/aws-lambda/schema.lua

```diff
@@ -20,11 +22,13 @@ return {
     },
     aws_key = {
       type = "string",
-      required = true,
     },
     aws_secret = {
       type = "string",
-      required = true,
+    },
+    use_ec2_iam_role = {
```

Comment:
Hmm indeed... I do agree with that reasoning. We kind of set this "unsaid" policy of enforcing migrations when introducing new values, but optional values would not benefit from it in any way, because they might be `nil` per definition already.

Alright then, this sounds good! Sorry for the noise!